hal/inc/concurrent_hal.h

```diff
@@ -132,4 +181,3 @@ void os_thread_scheduling(bool enabled, void* reserved);
 #endif
```

Comment:
You learn something new everyday! I never realized that was the case. I
must never have tried to compile a function with default arguments in ANSI
C 😁
On Sep 20, 2015 1:18 PM, "Matthew McGowan" <notifications@github.com> wrote:

> In hal/inc/concurrent_hal.h
> <https://github.com/spark/firmware/pull/614#discussion_r39932059>:
>
> > @@ -132,4 +181,3 @@ void os_thread_scheduling(bool enabled, void* reserved);
> >  #endif
>
> It's a C API - there are no default values in C. As soon as we raise the
> bar to having everything C++ then we can do this (and fix some other long
> standing issues, like having to use #defines for pin constants.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spark/firmware/pull/614/files#r39932059>.
>